code/controllers/subsystem/polling.dm

```diff
@@ -42,28 +45,32 @@ SUBSYSTEM_DEF(polling)
 	ignore_category = null,
 	flash_window = TRUE,
 	list/group = null,
-	pic_source,
+	alert_pic,
+	jump_target,
 	role_name_text,
 	list/custom_response_messages,
 	start_signed_up = FALSE,
+	amount_to_pick = 0,
+	chat_text_border_icon,
+	announce_chosen = TRUE
 )
-	RETURN_TYPE(/list/mob)
 	if(group.len == 0)
-		return list()
+		return
 	if(role && !role_name_text)
 		role_name_text = role
 	if(role_name_text && !question)
-		question = "Do you want to play as [full_capitalize(role_name_text)]?"
+		question = "Do you want to play as \a [span_notice(role_name_text)]?"
```

Comment:
Does this `\a` even work with the following macro there?

Reply:
nope, i did find a way to make it work my modifying role_name_text, but i decided to drop the \a because it may not always be appropriate